examples/examples/fmt-fmtspan.rs

```diff
@@ -0,0 +1,22 @@
+#![deny(rust_2018_idioms)]
+
+use tracing_subscriber::{fmt::format::FmtSpan, prelude::*, subscribe::CollectExt};
+#[path = "fmt/yak_shave.rs"]
+mod yak_shave;
+
+fn main() {
+    tracing_subscriber::registry()
+        .with(tracing_subscriber::fmt::timing::TimingSubscriber::default())
+        .with(tracing_subscriber::FmtSubscriber::new().with_span_events(FmtSpan::FULL))
```

Comment:
This is the main part I'd want to work better / get guidance on what's best for tracing. Perhaps this is the right thing to do. Maybe moving this into its own top level module, and providing something like the following would be useful?
```rust
tracing_subscriber::timing::subscriber()
```